datascience/predicates.py

```diff
@@ -0,0 +1,63 @@
+"""Predicate functions."""
```

Comment:
I think it'd be a good idea to also define the `__all__` variable for this file.